### ISSUE TITLE
Cannot purge partially installed app

### ISSUE THREAD

**Issue by madeofpeople** [kalabox/kalabox#416]
Had a docker issue on installing an app from pantheon. 
But now it hangs on purge.

```
$ cd site-name/
$ kbox uninstall -- --purge
info: DOCKER =>  Starting container kalabox_syncthing.
info: DOCKER =>  Container started. kalabox_syncthing
EVENTS =>  Error dispatching event listeners [pre-uninstall].
Error: HTTP code is 500 which indicates error: server error - Cannot start container 58d37e56e6ebf5333ed9955fa08b1b3facc449549ce43ab8ea989b98d7c2f30a: Could not apply volumes of non-existent container "kb_msdw-template_data".
```

**Comment by madeofpeople** [comment]
Have tried ```kbox down``` and ```kbox up``` after getting this issue to no avail.

**Comment by reynoldsalec** [comment]
I've had this issue in the past as well. The short-term fix is to restart kalabox (kbox down && kbox up) and re-install the app under a different name. However, long-term I'm guessing we'll need to improve the uninstall process to handle cases were there are "half-installed" problems like this.

**Comment by pirog** [comment]
AFAICT this seems to have been resolved if not in 0.10.1 then in the upcoming 0.10.2. Going to close for now.
